src/components/pages/batches.js

```diff
@@ -241,9 +339,17 @@ function Batches({ initialState, PageTemplate }) {
 
   taskTableColumns.push({
     title: t('taskStatus'),
-    dataIndex: 'processed',
+    dataIndex: 'localizedStatus',
     render: renderTaskStatus,
-    sorter: taskStatusSorter
+    sorter: (a, b) => a.localizedStatus.localeCompare(b.localizedStatus),
+    filters: [
+      { text: t('status_pending'), value: STATUS.pending },
+      { text: t('status_warning'), value: STATUS.warning },
+      { text: t('status_error'), value: STATUS.error },
+      { text: t('status_success'), value: STATUS.success }
+    ],
+    defaultFilteredValue: [STATUS.pending, STATUS.warning, STATUS.error],
```

Comment:
Thanks for the feedback. I already had the feeling that the filtering option might be very easy to be missed by the user and cause some head-scratching. I'll try to rework the UI so that it's hopefully clearer.

Reply:
I switched the (bearly noticable) filter to a simple checkbox

Reply:
Very nice and clear!